src/Printer.zig

```diff
@@ -3,14 +3,14 @@ const command = @import("./command.zig");
 
 const Self = @This();
 
-out: std.fs.File.Writer,
+file: std.fs.File,
```

Comment:
@bahodge Thanks for the contribution! Sorry for the delayed response - I was away last week.

I've moved to `std.Io.Writer` in the `zig-0.15` branch, which I'd prefer to stick with since it's the recommended approach. However, that branch currently doesn't compile with `zig-0.15.1`. I'll fix the compatibility issue and then merge it into `main`.